Makefile

```diff
@@ -286,6 +286,8 @@ generate-ci-config:
 
 generate-catalog:
 	./hack/generate/catalog.sh
+	./hack/generate/override-snapshot.sh \
+  	.konflux-release/
```

Comment:
Is this intended?

Reply:
it would be failing otherwise

Reply:
In #3132 I moved them to a separate target, because the override snapshot generation is anyhow "async" to the catalog update. 
But not blocking on this
/lgtm

Reply:
I see, we can do that too